ppa/archive/solr.py

```diff
@@ -48,7 +48,7 @@ class ArchiveSearchQuerySet(AliasedSolrQuerySet):
         "work_type_s": "work_type",
         "book_journal_s": "book_journal",
         "group_id_s": "group_id",
-        "cluster_id_s": "cluster_id",
+        "cluster_id_s": "cluster_id",    # <- will this conflict with property self.cluster_id_s? should I rename?
```

Comment:
this is for aliasing the solr field name, which uses dynamic field types, so that we can reference it with a meaningful name — I don't think it should conflict with the cluster id property